### PR TITLE
Add fuzzing tests for handlers

### LINKED ISSUE #326: Add fuzzing tests for handlers

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #326.

### CODE REVIEW

**Review by seeforschauer — APPROVED** [review]
nothing suspicious,  
and need an intro for fuzzing some day.

**Comment by AlekSi** [comment]
Oops, sorry, that one is not ready yet. I'm extracting smaller things from that PR for easier review.

File: internal/handlers/handler.go [hunk 1]
```diff
@@ -133,43 +136,70 @@ func (h *Handler) Handle(ctx context.Context, reqHeader *wire.MsgHeader, reqBody
 		fallthrough
 	default:
 		requests.WithLabelValues(command).Inc()
-		panic(fmt.Sprintf("unexpected OpCode %s", reqHeader.OpCode))
+
+		err = lazyerrors.Errorf("unexpected OpCode %s", reqHeader.OpCode)
```
Comment (seeforschauer) [inline internal/handlers/handler.go:140]: is the OpCode is unexpected, return here?

Comment (seeforschauer) [reply internal/handlers/handler.go:140]: no I see now - it's to be processed.
But the error is not handled somehow, it's overwritten by `	b, err := resBody.MarshalBinary()` later on

File: internal/handlers/handler.go [hunk 2]
```diff
@@ -133,43 +136,70 @@ func (h *Handler) Handle(ctx context.Context, reqHeader *wire.MsgHeader, reqBody
...
 	}
 
+	// set body for error
 	if err != nil {
```
Comment (AlekSi) [inline internal/handlers/handler.go:144]: there

File: internal/handlers/handler.go (hunk 1 above)
Comment (AlekSi) [reply internal/handlers/handler.go:140]: It is handled a few lines below

File: internal/handlers/handler.go (hunk 2 above)
Comment (seeforschauer) [reply internal/handlers/handler.go:144]: now I see.

File: internal/handlers/handler.go (hunk 1 above)
Comment (seeforschauer) [reply internal/handlers/handler.go:140]: ok

**Review by seeforschauer — APPROVED** [review]

**Pull request merged**